### PR TITLE
chore: ensure adminFacet for bldHolder is saved.

### LINKED ISSUE #7433: power to upgrade BLD mintHolder contract not saved; others?
## Describe the bug

While adding labels to calls to `startInstance()` (#7426) I discovered a couple where we don't save the `adminFacet` and hence can't upgrade. I fixed `economicCommittee` but not `mintHolder` and I'm not certain there are no other cases.

https://github.com/Agoric/agoric-sdk/blob/c832a55a303dbe2f1ae750c9f6a4874ea570998c/packages/inter-protocol/src/proposals/startPSM.js#L254

## Expected behavior

The `adminFacet` from every call to `startInstance` is available to later bootstrap actions for upgrade.

## Platform Environment
 - what version of the Agoric-SDK?

c832a55a303dbe2f1ae750c9f6a4874ea570998c

## Additional context

This is essentially the same bug as #6034, but I gather we prefer not to re-open issues.

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
closes: #7433

## Description

ensure  adminFacet for bldHolder is saved.

### Security Considerations

This ensures we can upgrade or recover.

### Scaling Considerations

None

### Documentation Considerations

None

### Testing Considerations

Trying to find relevant tests...

### CODE REVIEW

File: packages/vats/src/core/basic-behaviors.js [hunk 1]
```diff
@@ -696,11 +700,13 @@ export const BASIC_BOOTSTRAP_PERMITS = {
       bridgeManager: 'bridge',
       // TODO: re-org loadCriticalVat to be subject to permits
       loadCriticalVat: true,
+      startUpgradable: true,
       zoe: true,
     },
     produce: {
       bankManager: 'bank',
       bldIssuerKit: true,
+      bldMintHolderKit: true,
```
Comment (dckc) [inline packages/vats/src/core/basic-behaviors.js:708]: This requires an update to the bootstrap visualization snapshot.

In `packages/vats` run `yarn test test/test-boot-config.js --update-snapshots`

**Review by dckc — APPROVED** [review]
looks good, presuming the snapshot test is updated

File: packages/vats/src/core/basic-behaviors.js (hunk 1 above)
Comment (Chris-Hibbert) [reply packages/vats/src/core/basic-behaviors.js:708]: yep. found it.

**Comment by samsiegart** [comment]
Thank you for catching this one!

**Pull request merged**